draft/index.html

```diff
@@ -222,7 +217,8 @@ <h4>This Draft</h4>
           <ul>
             <li>Remove support for flyouts;</li>
             <li>The endpoints of various sub-services are no longer configurable, but determined by the root reconciliation endpoint;</li>
-            <li>The reconciliation queries only support passing a single <code>type</code> and the <code>type_strict</code> parameter was removed.</li>
+            <li>The reconciliation queries only support passing a single <code>type</code> and the <code>type_strict</code> parameter was removed;</li>
+            <li>Remove JSONP support.</li>
```

Comment:
```suggestion
            <li>The reconciliation queries only support passing a single <code>type</code> and the <code>type_strict</code> parameter was removed</li>
            <li>Remove JSONP support</li>
```
I'm not sure if there's a specific style guide being followed which requires this punctuation, but it seems odd to me. It's certainly not commonly used in American English. I would suggest removing all the terminal punctuation for the list entries. (Applies to the two list entries above as well)

Reply:
Yeah, I noticed that as well, but I figured that it's best to just keep following the established style.

Should I remove the punctuation in this PR? Make another PR? There are two more lists that use similar punctuation above this one (the ones for versions 0.1 and 0.2).

Reply:
Agreed, we can change the formatting of the list in another PR.